### PR TITLE
Add Sentry logging

### PR DESCRIPTION
**Pull request by bcongdon**
Add Raven / Sentry logging (only for backend right now)

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -202,11 +204,20 @@
         }
     },
     'handlers': {
+        'sentry': {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+            'tags': {'custom-tag': 'x'},
```
Comment (yuna9) [inline settings.py]: Probably want to change the tag here

File: settings.py [hunk 2]
```diff
@@ -202,11 +204,20 @@
...
+        },
         'mail_admins': {
             'level': 'ERROR',
             'filters': ['require_debug_false'],
             'class': 'django.utils.log.AdminEmailHandler'
         },
+        'console': {
```
Comment (yuna9) [inline settings.py:216]: Did you mean to add this also?

**Comment by yuna9** [comment]
:package: :ship: :moneybag:

CI is red though

**Comment by bcongdon** [comment]
Yeah, travis isn't happy because a new setting was added to local.py, but the local.py it's pulling from the tracker doesn't have the new config yet.

Should go green once the tracker PR gets merged

File: settings.py (hunk 1 above)
Comment (bcongdon) [reply settings.py]: Good catch

File: settings.py (hunk 2 above)
Comment (bcongdon) [reply settings.py:216]: Yea, it seems that raven/sentry need a dummy handler to point at

**Comment by Lange** [comment]
We don't actually have any CI script for this repo lol, I must have turned on Travis at some point. I just told travis to not build unless `.travis.yml` is present.

**Pull request merged**